plugins/modules/keycloak_realm.py

```diff
@@ -567,6 +567,144 @@
       - oauth2DevicePollingInterval
     type: int
     version_added: 11.2.0
+  web_authn_policy_rp_entity_name:
+    description:
+      - WebAuthn Relying Party Entity Name.
+    aliases:
+      - webAuthnPolicyRpEntityName
+    type: str
```

Comment:
```suggestion
    type: str
    version_added: 11.3.0
```
Same for the others.